Sources/Architecture/X86/HardwareBreakpointManager.cpp

```diff
@@ -54,6 +54,7 @@ ErrorCode HardwareBreakpointManager::enableLocation(Site const &site) {
 
   error = thread->writeDebugReg(idx, site.address);
   if (error != kSuccess) {
+    DS2LOG(Error, "Failed to write to debug address register");
```

Comment:
This commit is good except for the capitalization. I'll fix that nit and commit the patch.